evm/deploy-upgradable.js

```diff
@@ -95,7 +95,8 @@ function getUpgradeArgs(contractName, config) {
  * Deploy or upgrade an upgradable contract that's based on the init proxy pattern.
  */
 async function deploy(options, chain) {
-    const { env, artifactPath, contractName, privateKey, upgrade, verify } = options;
+    const { artifactPath, contractName, privateKey, upgrade, verifyEnv } = options;
```

Comment:
there is no `verifyEnv` option. Those are separate options and if verify is enabled, then verifyOptions can be created below

Reply:
I added verify as an option and create verifyOptions only if verify is enabled